pkg/doc/jose/encrypter_decrypter_test.go

```diff
@@ -159,6 +162,63 @@ func TestJWEEncryptRoundTrip(t *testing.T) {
 	}
 }
 
+func TestRoundTripInteropWithGoJose(t *testing.T) {
+	recECKeys, recKHs := createRecipients(t, 3)
+	gjRecipients := convertToGoJoseRecipients(t, recECKeys)
+
+	gjEncrypter, err := jose.NewMultiEncrypter(jose.A256GCM, gjRecipients, nil)
+	require.NoError(t, err)
+
+	pt := []byte("Test secret message")
+
+	// encrypt pt using go-jose encryption
+	gjJWEEncrypter, err := gjEncrypter.Encrypt(pt)
+	require.NoError(t, err)
+
+	// get go-jose serialized JWE
+	gjSerializedJWE := gjJWEEncrypter.FullSerialize()
+
+	// deserialize using local jose package
+	localJWE, err := Deserialize(gjSerializedJWE)
+	require.NoError(t, err)
+
+	for i, recKH := range recKHs {
+		recipientKH := recKH
+
+		t.Run(fmt.Sprintf("%d: Decrypting JWE message encrypted by go-jose test success", i), func(t *testing.T) {
+			jweDecrypter := NewJWEDecrypt(recipientKH)
+
+			var msg []byte
+
+			msg, err = jweDecrypter.Decrypt(localJWE)
+			require.NoError(t, err)
+			require.EqualValues(t, pt, msg)
+		})
+	}
+}
+
+func convertToGoJoseRecipients(t *testing.T, keys []ecdhessubtle.ECPublicKey) []jose.Recipient {
+	t.Helper()
+
+	var joseRecipients []jose.Recipient
+
+	for _, key := range keys {
+		c := subtle.GetCurve(key.Curve)
+		gjKey := jose.Recipient{
+			Algorithm: jose.ECDH_ES_A256KW, // for now, this is the only key wrapping algorithm supported
```

Comment:
Not sure if you need this comment... consider one day if we add another key wrapping algorithm... unless you remember to update this comment it'll stay there and then be misleading

Reply:
I'll remove it..

Reply:
removed